repository_service_tuf/cli/admin/import_targets.py

```diff
@@ -65,10 +77,12 @@ def _import_csv_to_rstuf(
 
     for csv_file in csv_files:
         console.print(f"Import status: Loading data from {csv_file}")
-        rstuf_db_data = _parse_csv_data(csv_file, succinct_roles)
+        rstuf_db_data = _parse_csv_data(
+            db_client, rstuf_target_roles, succinct_roles, csv_file
+        )
         console.print(f"Import status: Importing {csv_file} data")
         try:
-            db_client.execute(rstuf_table.insert(), rstuf_db_data)
+            db_client.execute(rstuf_target_files.insert(), rstuf_db_data)
```

Comment:
So, this will add a new record to the `rstuf_target_files` table and the db client will put the `rstuf_db_data` on its place?

Reply:
The `rstuf_db_data` is the data to be added to the DB, but by default, it is a Python dictionary where the key is the column name.

Reply:
I think I got what this line means.
We ask the `db` connection to execute a task or to `insert` what is given as a second argument or `rstuf_db_data`.
Am I correct?

Reply:
Yes, that is correct.